includes/base/widget-base.php

```diff
@@ -525,7 +525,7 @@ public function add_lightbox_data_to_image_link( $link_html, $id ) {
 	 * @access public
 	 *
 	 */
-	public function add_lightbox_data_attributes( $element, $id = null, $lightbox_setting_key = null, $group_id = null, $overwrite = false ) {
+	public function add_lightbox_data_attributes( $element, $id = null, $lightbox_setting_key = null, $group_id = null, $overwrite = false, $element_type = null ) {
```

Comment:
What will happen for 3rd party that didn't set it?

Reply:
Good question. Not sure. Will do some additional tests.